reflex/components/radix/themes/base.py

```diff
@@ -210,3 +212,61 @@ class RadixThemesColorModeProvider(Component):
     library = "/components/reflex/radix_themes_color_mode_provider.js"
     tag = "RadixThemesColorModeProvider"
     is_default = True
+
+
+class RadixThemesComponentPropsOverride(Component):
```

Comment:
maybe we can make this just a plain mixin class (i.e. not a subclass of Component)

and then override `__init__` to do the conversions before passing off to `super().__init__`.

That way we wouldn't need to redefine `create`.

The RadixThemesComponentPropsOverride would have to be named first in the list of parent classes though, to ensure an actual `Component` `__init__` didn't get called first.